obspy/io/segy/segy.py

```diff
@@ -116,6 +117,16 @@ def __init__(self, file=None, endian=None, textual_header_encoding=None,
         :param read_traces: Data traces will only be read if this is set to
             ``True``. The data will be completely ignored if this is set to
             ``False``.
+        :type map_header: list
+        :param non map_header: List of lists with headerwords that have been
```

Comment:
What's that 'non'? Is it marking not required param?

Reply:
No it's a mistake on my side.